docs/Introduction/tutorials/intermediates-tutorial.md

```diff
@@ -344,62 +342,45 @@ contract VRFD20 is VRFConsumerBase, ConfirmedOwner(msg.sender) {
   <a href="/docs/conceptual-overview/#what-is-remix" >What is Remix?</a>
 </div>
 
-You have now completed all necessary functions to generate randomness and assign the user a *Game of Thrones* house. We've added a few helper functions in there to make using the contract easier and more flexible. You can deploy and interact with the complete contract in Remix.
+You have now completed all necessary functions to generate randomness and assign the user a _Game of Thrones_ house. We've added a few helper functions in there to make using the contract easier and more flexible. You can deploy and interact with the complete contract in Remix.
 
-# 5. How do I deploy to testnet?
+## 5. How do I deploy to testnet?
 
 You will now deploy your completed contract. This deployment is slightly different than the example in the [Deploy Your First Contract](/docs/deploy-your-first-contract/) guide. In our case, you will have to pass in parameters to the constructor upon deployment.
 
 Once compiled, you'll see a dropdown menu that looks like this in the deploy pane:
 
 ![Remix contract selected](/files/intermediates-tutorial-01.png)
 
-Select the `VRFD20` contract or the name that you gave to your contract. You will deploy this contract on the Kovan test network.
-
-Click the caret arrow on the right hand side of **Deploy** to expand the parameter fields, and paste the following values in:
-
-- vrfCoordinator:   `0xdD3782915140c8f3b190B5D67eAc6dc5760C46E9`
-- LINK address:     `0xa36085f69e2889c224210f603d836748e7dc0088`
-- keyHash:          `0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4`
-- fee:              `100000000000000000`
+Select the `VRFD20` contract or the name that you gave to your contract. You will deploy this contract on the Rinkeby test network.
 
-![Remix contract parameters to deploy](/files/intermediates-tutorial-03.png)
+Click the caret arrow on the right hand side of **Deploy** to expand the parameter fields, and paste your subscription id.
 
-Then click the `transact` button.
+![Remix contract parameters to deploy](/files/intermediates-tutorial-02.png)
```

Comment:
You might want to scale some of the larger images down a little bit if you have time.